gto/git_utils.py

```diff
@@ -15,27 +16,22 @@ def wrapped_f(*args, **kwargs):
         kwargs = _turn_args_into_kwargs(args, kwargs)
 
         # noinspection PyTypeChecker
-        if isinstance(kwargs["repo"], str) and is_url_to_remote_repo(
+        if isinstance(kwargs["repo"], str) and is_url_of_remote_repo(
             repo=kwargs["repo"]
         ):
             try:
-                with TemporaryDirectory() as tmp_dir:
-                    logging.debug("create temporary directory %s", tmp_dir)
-                    # noinspection PyTypeChecker
-                    git_clone(repo=kwargs["repo"], dir=tmp_dir)
+                # noinspection PyTypeChecker
```

Comment:
Haven't seen this anywhere earlier. This is something to suppress PyCharm suggestions?

Reply:
Yes, if it bothers you I can remove it

Reply:
```suggestion
```